File: gc-commons/gc-common/src/main/java/com/gc/common/message/ResultCodeEnum.java
package com.gc.common.message;

import lombok.Getter;

@Getter
public enum ResultCodeEnum {

    /**
     * 成功
     */
    SUCCESS(200, "成功"),
    /**
     * 失败
     */
    FAILURE(500, "失败");

    private Integer code;

    private String msg;

    ResultCodeEnum(Integer code, String msg) {
        this.code = code;
        this.msg = msg;
    }
}

File: smart-commons/smart-common/src/main/kotlin/com/smart/common/message/ResultCodeEnum.kt
package com.smart.common.message

/**
 *
 * @author ming
 * 2019/6/12 上午10:12
 */
enum class ResultCodeEnum constructor(val code: Int, val msg: String) {
    /**
     * 成功
     */
    SUCCESS(200, "成功"),
    /**
     * 失败
     */
    FAILURE(500, "失败")

}